src/pages/courts/index.tsx

```diff
@@ -9,39 +9,48 @@ import {
   IconButton,
   Button,
   useToast,
+  Text,
 } from "@chakra-ui/react";
-import { useEffect, useState } from "react";
+import React from "react";
```

Comment:
I think we might not need to import React in Nextjs project as it should automatically be imported. Give it a try.

Reply:
ok, it has been removed!